invision-app/app/tests/reducer.test.js

```diff
@@ -0,0 +1,63 @@
+import {moviesReducer } from '../store/reducer';
+import {
+    FETCH_MOVIES,
+    CREATE_MOVIE,
+    DELETE_MOVIE,
+    UPDATE_MOVIE,
+    SORT_BY,
+    FILTER_BY_GENRE,
+    SHOW_DESCRIPTION_MOVIE,
+    SEARCH_MOVIES
+} from '../store/types';
+import moviesData  from '../model/moviesData.json'
+
+describe('Movie Reducer', () => {
+    const initValue = {
```

Comment:
You already have **initialState** in the reducer file - **app/store/reducer.js**. You can export this initial store object and use it where you want. Please, avoid code duplication.